### ISSUE TITLE
As a repo owner I need to be able to create changelogs

### ISSUE THREAD

**Issue by aaronmallen** [JustShipIt/docQment#4]
Model will need the following fields:
* `change_type` (this can be changed, added, fixed)
* `version`
* `release_status` (whether or not the new changes are in master or not)
* `change` (what was changed)

**Comment by aaronmallen** [comment]
The suggested workflow for v1.0.0:

1. Developer submits a PR and follows [guide lines](https://github.com/JustShipIt/docQment/wiki/Creating-Changelog-Data) for changelog documentation.
1. A live preview of the changelog is generated.
1. Developer edits comments/description on PR
1. Changelog is saved to database once PR is merged into master.
     * The changelog version will be determined by repo tags.  We will need to look for the most recent tag after changelog is merged.
    
  * This would look like this:
      * commit-6
      * PR-3 merged into master
      * commit-4 -tag v1.0.1
      * PR-2 merged into master
      * commit-2 -tag v.1.0.0
      * PR-1 merged into master
  in this example the comments/description for PR-1 would be in v1.0.0, PR-2 would be in v1.0.1, PR-3 would be "unreleased"

@wasi would you care to chip in on this? Feel free to suggest modifications to this workflow or ask questions if it doesn't make any sense.
